config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Aeronbroker/Aeron",
+            "Aeronbroker/NEConfMan"
+        ],
+        "docker": [
+            "fiware/iotbroker"
+        ]
+    },
+    {
+        "enabler": "IoT Discovery",
+        "catalogue": "iot-discovery",
+        "academy": "IoT Discovery",
+        "readthedocs": "fiware-iot-discovery-ngsi9",
+        "helpdesk": "IoT Discovery",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "UniSurreyIoT/fiware-iot-discovery-s2w",
+            "UniSurreyIoT/fiware-iot-discovery-ngsi9"
+        ],
+        "docker": [
+            "fiware/fiware-iot-discovery-ngsi9"
+        ]
+    },
+    {
+        "enabler": "Keyrock - Identity Manager",
+        "catalogue": "identity-management-keyrock",
+        "academy": "KeyRock",
+        "readthedocs": "fiware-idm",
+        "helpdesk": "KeyRock",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-idm"
+        ],
+        "docker": [
+            "fiware/idm"
+        ]
+    },
+    {
+        "enabler": "Kiara",
+        "catalogue": "kiara-advanced-middleware",
+        "academy": "KIARA Advanced Middleware",
+        "readthedocs": "",
+        "helpdesk": "Kiara",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "FIWARE-Middleware/KIARA"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Knowage",
+        "catalogue": "data-visualization-knowage",
+        "academy": "",
+        "readthedocs": "knowage",
+        "helpdesk": "Knowage",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "KnowageLabs/Knowage-Documentation",
+            "KnowageLabs/Knowage-Server",
+            "KnowageLabs/Knowage-Server-Chef",
+            "KnowageLabs/Knowage-Server-Docker"
+        ],
+        "docker": [
+            "knowagelabs/knowage-server-docker"
+        ]
+    },
+    {
+        "enabler": "Kurento",
+        "catalogue": "kurento",
+        "academy": "Kurento",
+        "readthedocs": "doc-kurento",
+        "helpdesk": "Kurento",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Kurento/doc-fiware-readthedocs",
+            "Kurento/kurento-media-server",
+            "Kurento/kurento-docker"
+        ],
+        "docker": [
+            "fiware/stream-oriented-kurento"
+        ]
+    },
+    {
+        "enabler": "MR CoAP - Protocol Adapter",
+        "catalogue": "protocol-adapter-mr-coap",
+        "academy": "Protocol Adapter",
+        "readthedocs": "",
+        "helpdesk": "",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": []
+    },
+    {
+        "enabler": "Micro-XRCE-DDS",
```

Comment:
"enabler": "Micro XRCE-DDS"

please check catalogue variable here...

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585